cmd/defradb/main.go

```diff
@@ -12,19 +12,12 @@
 package main
 
 import (
-	"context"
-	"os"
-
 	"github.com/sourcenetwork/defradb/cli"
 	"github.com/sourcenetwork/defradb/config"
 )
 
 // Execute adds all child commands to the root command and sets flags appropriately.
 func main() {
-	cfg := config.DefaultConfig()
-	ctx := context.Background()
-	defraCmd := cli.NewDefraCommand(cfg)
-	if err := defraCmd.Execute(ctx); err != nil {
-		os.Exit(1)
-	}
+	defraCmd := cli.NewDefraCommand(config.DefaultConfig())
+	defraCmd.Execute() //nolint:errcheck
```

Comment:
question: why?

Reply:
I've updated it to exit with a non-zero status code on error. The errors are already logged by cobra.

Reply:
Can you please add a code-comment documenting why you think this error is fine to discard.